src/sec_newton.mbx

```diff
@@ -558,222 +557,577 @@
         </webwork>
       </exercise>
 
-
-
+<!--Exercise 4 sine function-->
       <exercise>
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+            <var name="$f">
+              <static>\sin(x)</static>
+            </var>
+            <var name="$x[0]">
+              <static>1</static>
+            </var>
+            <var name="$x[1]">
+              <static>-0.55740772</static>
+            </var>
+            <var name="$x[2]">
+              <static>0.065936452</static>
+            </var>
+            <var name="$x[3]">
+              <static>-0.000095721919</static>
+            </var>
+            <var name="$x[4]">
+              <static>2.9235662*10^-13</static>
```

Comment:
Does this apply to x^17-2x^13-10x^8+10 as well? As in, x^{17}-2x^{13}-10x^{8}+10? (Inside the static tags.)